lib/util/parse-connection.js

```diff
@@ -1,20 +1,32 @@
-const url = require('url');
+const { URL } = require('url');
```

Comment:
Since node 10 URL is available globally like in browsers. This require can be removed.

Reply:
done